<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="gmps_ip" default="192.168.1.10"/>
  <arg name="gmps_port" default="9501"/>
  <arg name="gmpssta_timeout" default="0.5"/>

  <group>
    <!--push-ros-namespace namespace="gmps"/-->

    <node pkg="gmps_driver" exec="gmps_tcp_receiver" name="gmps_tcp_receiver" output="log" >
      <param name="gmps_ip" value="$(var gmps_ip)"/>
      <param name="gmps_port" value="$(var gmps_port)"/>
      <param name="gmpssta_timeout" value="$(var gmpssta_timeout)"/>
    </node>
  </group>
</launch>
